packages/shared/src/graphql/squads.ts

```diff
@@ -552,44 +551,6 @@ export const SQUAD_COMMENT_JOIN_BANNER_KEY = generateStorageKey(
   StorageTopic.Squad,
   'comment_join_banner',
 );
-
-export const SUBMIT_SQUAD_FOR_REVIEW_MUTATION = gql`
-  mutation SubmitSquadForReview($sourceId: ID!) {
-    submitSquadForReview(sourceId: $sourceId) {
-      status
-    }
-  }
-`;
-
-interface SubmitSquadForReviewOutput {
-  submitSquadForReview: PublicSquadRequest;
-}
-
-export async function submitSquadForReview(
-  sourceId: string,
-): Promise<PublicSquadRequest> {
-  const data = await gqlClient.request<SubmitSquadForReviewOutput>(
-    SUBMIT_SQUAD_FOR_REVIEW_MUTATION,
-    { sourceId },
-  );
-  return data.submitSquadForReview;
-}
-
-interface GetPublicSquadRequestsProps {
-  sourceId: string;
-  first?: number;
-}
-
-export const getPublicSquadRequests = async (
-  params: GetPublicSquadRequestsProps,
-): Promise<Connection<PublicSquadRequest>> => {
-  const res = await gqlClient.request<{
-    requests: Connection<PublicSquadRequest>;
-  }>(PUBLIC_SQUAD_REQUESTS, params);
-
-  return res.requests;
-};
-
```

Comment:
Removed those functions because where no longer used (we do not have public squad request any more)